bin/stronghold-cli

```diff
@@ -157,9 +156,18 @@ opts.each do |opt, arg|
     }
   when '--env'
     mode_ify(opt)
-    from, to = arg.split(?:)
-    die(-6, "Wrong format to env, use like --env=hashpath:VAR") if from.nil? || to.nil?
-    ENV[to] = extract(valid_tree(), from).to_s
+    if arg.empty?
+      # Export all the config
+      tree = valid_tree()
+      tree.each_pair do |k, v|
+        ENV[k] = v.to_s
+      end
+      ENV['STRONGHOLD_ETAG'] = tree.etag
```

Comment:
Thanks. Not exactly critical, but it might hint to people that they won't find the `STRONGHOLD_ETAG` key in the actual data hosted in stronghold, for example.